### PR TITLE
Briang/app key renaming

### PR DESCRIPTION
**Pull request by spartan218**

### CODE REVIEW

File: b2/console_tool.py [hunk 1]
```diff
@@ -278,6 +278,7 @@ def run(self, args):
                 )
                 self.api.account_info.clear()
                 return 1
+            self.api.authorize_account(realm, args.applicationKeyId, args.applicationKey)
```
Comment (spartan218) [inline b2/console_tool.py]: This is a bug, not sure why it got added. 
Will remove it soon.

Comment (spartan218) [reply b2/console_tool.py]: This is fixed in https://github.com/Backblaze/B2_Command_Line_Tool/pull/570/commits/4e785fd1b0d793645668c62e9a3be835381c9311

File: test/test_console_tool.py [hunk 2]
```diff
@@ -685,7 +685,7 @@ def test_get_account_info(self):
         self._authorize_account()
         expected_stdout = '''
         {{
-            "accountAuthToken": "auth_token_0",
+            "accountAuthToken": "auth_token_1",
             "accountId": "{account_id}",
```
Comment (spartan218) [inline test/test_console_tool.py]: To be honest I dont know why the auth_token numbers were off, but my unit tests fail without this change.
If you know something I don't let me know, I didn't dig too deep to figure out why.

Comment (spartan218) [reply test/test_console_tool.py]: Apparently these are wrong this time.. will revert soon once my pre-commit.sh works again.

File: README.md [hunk 3]
```diff
@@ -103,13 +103,16 @@ Changes:
 
 * Most of the code moved to b2sdk repository and package
 * Deprecation warning added for imports of sdk classes from cli package
+* Renaming accountId for authentication to application key Id
```
Comment (bwbeach) [inline README.md]: This will be in release 1.4.2, not 1.4.0.

File: README.md [hunk 4]
```diff
@@ -97,16 +97,21 @@ For advanced users, a hidden option `--logConfig <filename.ini>` can be used to
 
 # Release History
 
-## 1.4.0 (April 6, 2019)
+## 1.4.0 (April 12, 2019)
```
Comment (spartan218) [inline README.md]: No version change since Pawel said he will piggy back off changes and bump the version.

Comment (bwbeach) [reply README.md]: Oh.  I see.  1.4.0 isn't released yet.  Got it.

I think I'd find it clearer not to have a date there yet for un-released versions.  But no need to change that now.

Comment (ppolewicz) [reply README.md]: @bwbeach that's a very good point

**Pull request merged**